app/controllers/general.php

```diff
@@ -288,6 +288,9 @@ function router(App $utopia, Database $dbForConsole, callable $getProjectDB, Swo
             $execution->setAttribute('logs', $executionResponse['logs']);
             $execution->setAttribute('errors', $executionResponse['errors']);
             $execution->setAttribute('duration', $executionResponse['duration']);
+
+        } catch (AppwriteException $ex) {
+            throw $ex;
```

Comment:
Whats the need for this ?

Reply:
The `createExecution` method is called inside this `try` block, and therefore without this the `AppwriteException` thrown within will be caught by the existing `catch` block below.

If the `AppwriteException is `caught`, we show an error page.